src/web/layouts/lib/stickiness.tsx

```diff
@@ -1,30 +1,29 @@
 import { css } from '@emotion/react';
 
-import { border } from '@guardian/src-foundations/palette';
-
 import { getZIndex, getZIndexImportant } from '@frontend/web/lib/getZIndex';
 
 type Props = {
 	children?: React.ReactNode;
 };
 
-type StuckProps = Props & {
+type StuckProps = {
+	children?: React.ReactNode;
 	zIndex?: string;
 };
 
-// The advert is stuck to the top of the container as we scroll
-// until we hit the bottom of the wrapper that contains
-// the top banner and the header/navigation
-// We apply sticky positioning and z-indexes, the stickAdWrapper and headerWrapper
-// classes are tightly coupled.
-const stickyAdWrapper = (zIndex = 'stickyAdWrapper') => css`
-	background-color: white;
-	border-bottom: 0.0625rem solid ${border.secondary};
+const stickyStyles = css`
 	position: sticky;
 	top: 0;
+`;
+
+const addZindex = (zIndex = 'stickyAdWrapper') => css`
 	${getZIndex(zIndex)}
 `;
 
+const whiteBackground = css`
+	background-color: white;
+`;
```

Comment:
By breaking unrelated css out into separate chunks we make the code more scalable for future use cases - this kind of approach might have prevented the issue this PR deals with where unrelated css got carried through when it wasn't needed.